style/header.css

```diff
@@ -1,3 +1,7 @@
+/* body {
```

Comment:
usually, when you comment out code and leave it, you should leave an actual comment explaining why this code is commented out.
this practice improves code readability and helps you when you coming back to the code after a long period of time